src/components/Heroes/columns.jsx

```diff
@@ -23,72 +22,72 @@ export default {
     displayName: strings.hero_pick_ban_rate,
     field: 'pickBanRatePro',
     sortFn: true,
-    displayFn: (row, col, field) => percentileDisplay(field, row.matchCountPro),
+    percentBars: row => decimalToCount(row.pickBanRatePro, row.matchCountPro),
```

Comment:
They both get their values from sortFn or row[column.field]. Their functions return the small value that appears next to the main value